client/extensions/wp-job-manager/state/reducer.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Internal dependencies
+ */
+import { combineReducers } from 'state/utils';
+import settings from './settings/reducer';
+
+export default combineReducers( {
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15299/commits/1c66c0fe47182fca19f03dfe0664a4f4d0b17524.